src/actors/stdin.rs

```diff
@@ -61,7 +62,7 @@ impl StdInLines {
     async fn execute_command(&mut self, cmd: BossCommand) -> impl std::fmt::Display {
 
         match cmd {
-            BossCommand::SendCan { id, message, cycletime } => self.sender.send_can_message(id, message, cycletime).await,
+            BossCommand::SendCan { id: _ , message: _, cycletime: _ } => self.sender.send_can_message(123, 123, 0).await,

```

Comment:
the hardcoded values should not be here

Reply:
True, thanks for catching it and enforcing clean code on the project.

will address this.